### PR TITLE
add 67 add-binary

### PR DESCRIPTION
**Pull request by kobakobashu**
time : 15m

### CODE REVIEW

**Comment by kobakobashu** [comment]
Please check my code !

File: 67.add-binary.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#
+
+# @lc code=start
+class Solution:
+    def addBinary(self, a: str, b: str) -> str:
+        """
+        time: O(1)
+        space: O(1)
+        """
+        output_10 = int(a,2) + int(b,2)
```
Comment (ryuji0123) [inline 67.add-binary.py:14]: output_10 is ambiguous. Also, you don't have to initialize this variable.

```py
return bin(int(a,2) + int(b,2)).replace('0b', '')
```

File: 67.add-binary.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# @lc app=leetcode id=67 lang=python3
+#
+# [67] Add Binary
+#
+
+# @lc code=start
+class Solution:
+    def addBinary(self, a: str, b: str) -> str:
+        """
+        time: O(1)
```
Comment (ryuji0123) [inline 67.add-binary.py:11]: time complexity of int(str_1) is O(len(str_1))

**Review by ryuji0123 — CHANGES_REQUESTED** [review]

**Comment by kobakobashu** [comment]
I fixed my code!  Please check it!

**Review by ryuji0123 — APPROVED** [review]
LGTM

**Pull request merged**